<launch>
    <!-- projection node -->
    <node   pkg="segmentation_filtered_icp"
            type="cloud_filtering_node"
            name="cloud_filtering_node"
            output="screen">
        <param name="cloud_topic" value="/points2" />
        <param name="cloud_frame" value="rgb_camera_link" />
        <rosparam param="image_topics">
          - '/rgb/bg_fg_img'
        </rosparam>
        <rosparam param="image_frames">
          - rgb_camera_link
        </rosparam>
        <rosparam param="caminfo_topics">
          - /rgb/camera_info
        </rosparam>
        <param name="filtered_pc_topic" value="/points_filtered" />
        <param name="density_threshold" value="150" />
    </node>
</launch>